library/src/main/java/com/leinardi/android/speeddial/FabWithLabelView.java

```diff
@@ -179,21 +183,32 @@ public void setOnActionSelectedListener(OnActionSelectedListener listener) {
             getFab().setOnClickListener(new OnClickListener() {
                 @Override
                 public void onClick(View view) {
-                    mOnActionSelectedListener.onActionSelected(getSpeedDialActionItem());
+                    SpeedDialActionItem speedDialActionItem = getSpeedDialActionItem();
+                    if (mOnActionSelectedListener != null
+                            && speedDialActionItem != null) {
+                        mOnActionSelectedListener.onActionSelected(speedDialActionItem);
+                    }
                 }
             });
             getLabelBackground().setOnClickListener(new OnClickListener() {
                 @Override
                 public void onClick(View v) {
-                    if (getSpeedDialActionItem().isLabelClickable() && isLabelEnable()) {
-                        mOnActionSelectedListener.onActionSelected(getSpeedDialActionItem());
+                    SpeedDialActionItem speedDialActionItem = getSpeedDialActionItem();
+                    if (mOnActionSelectedListener != null
+                            && speedDialActionItem != null
+                            && speedDialActionItem.isLabelClickable()
+                            && isLabelEnable()) {
+                        mOnActionSelectedListener.onActionSelected(speedDialActionItem);
                     }
                 }
             });
-        } else {
+        } else
+
+        {
```

Comment:
**Reporter**: Checkstyle
**Rule**: com.puppycrawl.tools.checkstyle.checks.blocks.LeftCurlyCheck
**Severity**: ERROR
**File**: library/src/main/java/com/leinardi/android/speeddial/FabWithLabelView.java L207

'{' at column 9 should be on the previous line.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<-19697986>*